src/main/kotlin/io/github/loxygen/aimlessbot/lib/client/Client.kt

```diff
@@ -1,92 +1,70 @@
 package io.github.loxygen.aimlessbot.lib.client
 
-import io.github.loxygen.aimlessbot.cmds.OoooohShiiit
-import io.github.loxygen.aimlessbot.lib.commands.CommandExecutor
-import io.github.loxygen.aimlessbot.cmds.Ping
-import io.github.loxygen.aimlessbot.lib.commands.CommandResult
+import io.github.loxygen.aimlessbot.lib.commands.CommandManager
 import net.dv8tion.jda.api.JDA
 import net.dv8tion.jda.api.JDABuilder
 import net.dv8tion.jda.api.entities.Activity
 import net.dv8tion.jda.api.events.ReadyEvent
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent
 import net.dv8tion.jda.api.hooks.ListenerAdapter
+import kotlin.system.exitProcess
 
 /**
  * Discordのクライアント。
  */
-object Client : ListenerAdapter(){
+object Client : ListenerAdapter() {
 
-    /**
-     * Prefix付きのコマンドを実行するオブジェクトたち
-     */
-    val commandExecutors: List<CommandExecutor> = listOf(
-        Ping
-    )
-    /**
-     * プレフィックスがなくても発火するコマンドを実行するオブジェクトたち
-     */
-    val prefixlessCommandExecutor: List<CommandExecutor> = listOf(
-        OoooohShiiit
-    )
+   /**
+    * JDA実体
+    */
+   private lateinit var discordClient: JDA
 
-    /**
-     * コマンドの接頭辞
-     */
-    val PREFIX = "//"
-    /**
-     * JDA実体
-     */
-    private lateinit var discordClient: JDA
+   /**
+    * Botではあるけど弾かないユーザーのID
+    */
+   private lateinit var userIdsWhiteList: List<Long>
 
-    /**
-     * コマンドを実行する
-     * @param token Discordのトークン
-     */
-    fun launch(token: String) {
+   /**
+    * ログを出力するチャンネル。(デバッグ等で)
+    */
+   private var loggingChannelId: Long = -1
 
-        val jdaBuilder = JDABuilder.createDefault(token)
-        jdaBuilder.setActivity(Activity.of(Activity.ActivityType.DEFAULT, "hennlo world"))
-        jdaBuilder.addEventListeners(Client)
-        discordClient = jdaBuilder.build()
+   /**
+    * コマンドを実行する
+    * @param token Discordのトークン
+    */
+   fun launch(token: String, clientSettingInfo: ClientSettingInfo) {
 
-    }
+      val jdaBuilder = JDABuilder.createDefault(token)
+      jdaBuilder.setActivity(Activity.of(Activity.ActivityType.DEFAULT, "hennlo world"))
+      jdaBuilder.addEventListeners(Client)
+      discordClient = jdaBuilder.build()
```

Comment:
メソッドチェーンするかapply使いませんか
```suggestion
      discordClient = JDABuilder.createDefault(token)
      	.setActivity(Activity.of(Activity.ActivityType.DEFAULT, "hennlo world"))
      	.addEventListeners(Client)
      	.build()
```